tests/test_scripts.py

```diff
@@ -2693,6 +2693,17 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting():
```

Comment:
@SherineAwad, a few comments --

first, the test name should be something like 'test_counting_load_bigcount' - test_counting is too generic!

second, please create the 'count.ct.gz' file in this test, from a sequence file that is in test-data, rather than committing the count.ct.gz file to the repository directly for the test to use.  There are a couple reasons for this, but the main one is that if we update the counting table file format, this test will break :).

third, note that there is no guarantee in the test currently that bigcounts are loaded correctly - what you're really testing is that the count.ct.gz file can be loaded by load_counting_hash without a complaint.  That's great to test, but it's not testing anything about bigcounts particularly.  To test bigcount save/load, you should create a saved table from a FQ file that has k-mers with counts over 255; then load that in, and test to see if the k-mer count > 255 is retained.  A particularly easy way to do this would be to create a k-mer table with a ksize of 2 from one of our larger files in test-data, and then find a 2-mer that occurs a lot; some of the *abund* in tests/test-data/ might be a good place to look.

fourth, the advice about the assert function was, I think, incorrect.  The 'load' function raises an IOError if it can't load the file; this is a fine exception to break the test if load fails.  You can eliminate the try/except/assert that's there right now and the test will still fail properly if .load(...) fails.

This might be useful background reading: http://khmer.readthedocs.org/en/latest/dev/a-quick-guide-to-testing.html